radsat-sk/operation/crypt/encrypt.c

```diff
@@ -0,0 +1,58 @@
+#include "encrypt.h"
+
+int getPaddedBufSize(int plaintextLen);
+void padBuffer(uint8_t* buff, uint8_t* paddedBuff, int buffLen, int newBuffLen);
+void encrypt(uint8_t* plaintext, int plaintextLen);
+
+
+/**
+ * @brief Gets the next multiple of 16 from the size of the input buffer
+ * @header "comms-processing/crypt/encrypt.h"
+ * @param plaintextLen - The size of the unpadded buffer containing data to be encrypted.
+ * @pre need to have a variable storing the size of the plaintext buffer
```

Comment:
don't include the @pre tag if it's not necessary (in this case I would say it's not)